### DISCUSSION TITLE
[Popover] Popover.Content same width as Popover.Trigger

### DISCUSSION THREAD

**Discussion by SombreroElGringo** [radix-ui/primitives#1791] [Help] [answered]
Hello,

I would like to know how we can achieve this with the Popover component, let's say we have a Popover.Trigger using the full width available and when the Popover.Content is opened it should be the same width as the Popover.Trigger.

I would like to replicate the behavior of the [Select component](https://www.radix-ui.com/docs/primitives/components/select) popover where the width of the popover content is the same as the trigger.

Please if you have an example or suggestion to share that would be very helpful. Thanks!

PS: I just started with @radix-ui and for the moment everything is great 👍

**Comment by benoitgrelard** [comment, marked as answer]
Hey @SombreroElGringo,

Here's one way you could achieve this: https://codesandbox.io/s/popover-trigger-width-c7ttps?file=/App.js

**Reply by SombreroElGringo** [reply]
Thank you very much!
